great_expectations/checkpoint/util.py

```diff
@@ -225,7 +225,11 @@ def get_runtime_batch_request(
     if validation_batch_request is None:
         validation_batch_request = {}
 
-    runtime_batch_request_dict: dict = copy.deepcopy(validation_batch_request)
+    if isinstance(validation_batch_request, BatchRequest):
+        runtime_batch_request_dict: dict = validation_batch_request.to_json_dict()
```

Comment:
I believe this `BatchRequest` already runs through `convert_to_json_serializable` due to [PR #3349](https://github.com/great-expectations/great_expectations/pull/3349) and therefore this is correct.